### ISSUE TITLE
Error Opening Artisans Consortium Payout (loot container) addon v.1.7.0

### ISSUE THREAD

**Issue by arcadepro** [Mctalian/RPGLootFeed#127]
````1x ...y/LootDisplayFrame/LootDisplayRow/LootDisplayRow.lua:266: attempt to call method 'SetTexture' (a nil value)

==== Addon Info RPGLootFeed v1.7.0-1-c1e65b7 ====

Log traces related to ItemLoot
-------------------------------------------------
[06:12:07]{I}(WOW)[ITEM]: CHAT_MSG_LOOT You receive item: |Hitem:210931::::::::80:73::::1:38:2:::::[Bismuth |A:Professions-ChatIcon-Quality-Tier2:17:23::1|a]x6
[06:12:04]{I}(WOW)[ITEM]: CHAT_MSG_LOOT You receive item: |Hitem:227713::::::::80:73::11:::::::[Artisan's Consortium Payout]
[06:12:03]{I}(WOW)[ITEM]: CHAT_MSG_LOOT You receive item: |Hitem:210814::::::::80:73:::::::::[Artisan's Acuity]x10
[05:27:16]{I}(RLF)[ITEM]: |Hitem:222486::::::::80:73::13:3:10827:10830:9632:4:28:2734:29:80:38:8:40:2193:::::[Proficient Blacksmith's Hammer |A:Professions-ChatIcon-Quality-Tier5:17:17::1|a] x1 (tot: 1) [222486]
[05:27:16]{I}(WOW)[ITEM]: CHAT_MSG_LOOT You create: |Hitem:222486::::::::80:73::13:3:10827:10830:9632:4:28:2734:29:80:38:8:40:2193:::::[Proficient Blacksmith's Hammer |A:Professions-ChatIcon-Quality-Tier5:17:17::1|a].
[05:27:01]{I}(RLF)[ITEM]: |Hitem:222482::::::::80:73::13:3:10827:10830:9632:4:28:2734:29:79:38:8:40:2189:::::[Proficient Skinning Knife |A:Professions-ChatIcon-Quality-Tier5:17:17::1|a] x1 (tot: 1) [222482]
[05:27:01]{I}(WOW)[ITEM]: CHAT_MSG_LOOT You create: |Hitem:222482::::::::80:73::13:3:10827:10830:9632:4:28:2734:29:79:38:8:40:2189:::::[Proficient Skinning Knife |A:Professions-ChatIcon-Quality-Tier5:17:17::1|a].
[05:26:50]{I}(RLF)[ITEM]: |Hitem:222480::::::::80:73::13:3:10827:10830:9632:4:28:2734:29:77:38:8:40:2187:::::[Proficient Sickle |A:Professions-ChatIcon-Quality-Tier5:17:17::1|a] x1 (tot: 1) [222480]
[05:26:50]{I}(WOW)[ITEM]: CHAT_MSG_LOOT You create: |Hitem:222480::::::::80:73::13:3:10827:10830:9632:4:28:2734:29:77:38:8:40:2187:::::[Proficient Sickle |A:Professions-ChatIcon-Quality-Tier5:17:17::1|a].
[05:26:20]{I}(RLF)[ITEM]: |Hitem:222418::::::::80:73::::2:38:2:40:2136:::::[Core Alloy |A:Professions-ChatIcon-Quality-Tier2:17:23::1|a] x7 (tot: 7 (+1)) ~UPDATE~ [222418]
-------------------------------------------------

Please report this issue @ github: McTalian/RPGLootFeed


[string "@RPGLootFeed/LootDisplay/LootDisplayFrame/LootDisplayRow/LootDisplayRow.lua"]:266: in function `UpdateIcon'
[string "@RPGLootFeed/LootDisplay/LootDisplay.lua"]:120: in function <...rfaceRPGLootFeed/LootDisplay/LootDisplay.lua:80>
[string "@RPGLootFeed/LootDisplay/LootDisplay.lua"]:76: in function `ShowLoot'
[string "@RPGLootFeed/Features/LootDisplayProperties.lua"]:34: in function `Show'
[string "@RPGLootFeed/Features/ItemLoot.lua"]:118: in function <RPGLootFeed/Features/ItemLoot.lua:110>
[string "=[C]"]: ?
[string "@RPGLootFeed/Core.lua"]:48: in function `fn'
[string "@RPGLootFeed/Features/ItemLoot.lua"]:139: in function `?'
[string "@BigWigs/Libs/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:109: in function <...igs/Libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:109>
[string "=[C]"]: ?
[string "@BigWigs/Libs/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:19: in function <...igs/Libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:15>
[string "@BigWigs/Libs/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:54: in function `Fire'
[string "@HandyNotes/Libs/AceEvent-3.0-4/AceEvent-3.0.lua"]:120: in function <HandyNotes/Libs/AceEvent-3.0/AceEvent-3.0.lua:119>

Locals:
self = Frame {
 RightBorder = LootDisplayFrameRightBorder {
 }
 icon = 5728051
 _prev = Frame {
 }
 amount = 6
 Background = Texture {
 }
 Icon = Button {
 }
 TopBorder = LootDisplayFrameTopBorder {
 }
 link = "|cffffffff|Hitem:210931::::::::80:73::::1:38:2:::::|h[Bismuth |A:Professions-ChatIcon-Quality-Tier2:17:23::1|a]|h|r"
 BottomBorder = LootDisplayFrameBottomBorder {
 }
 key = "210931"
 HighlightAnimation = <table> {
 }
 AmountText = FontString {
 }
 LeftBorder = LootDisplayFrameLeftBorder {
 }
 FadeOutAnimation = AnimationGroup {
 }
}
key = "210931"
icon = 5728051
quality = nil
(*temporary) = nil
(*temporary) = Button {
 IconOverlay = Texture {
 }
 icon = LootDisplayFrameIconTexture {
 }
 itemContextMatchResult = 3
 searchOverlay = LootDisplayFrameSearchOverlay {
 }
 PushedTexture = Texture {
 }
 NormalTexture = LootDisplayFrameNormalTexture {
 }
 IconOverlay2 = Texture {
 }
 IconBorder = Texture {
 }
 HighlightTexture = Texture {
 }
 Stock = LootDisplayFrameStock {
 }
 ItemContextOverlay = Texture {
 }
 showMatchHighlight = true
 Count = LootDisplayFrameCount {
 }
}
(*temporary) = 5728051
(*temporary) = "attempt to call method 'SetTexture' (a nil value)"
Masque = nil
iconGroup = nil
````

**Comment by Mctalian** [comment]
Thanks for the report! I see the issue. I'll get a fix up as soon as I get to my PC

**Comment by Mctalian** [comment]
Thanks for your patience, should be fixed in v1.7.1!
